.github/workflows/tag-and-release.yml

```diff
@@ -0,0 +1,55 @@
+---
+name: Tag & Release
+'on':
+  push:
+    branches:
+      - "stackhpc/xena"
+      - "stackhpc/wallaby"
+      - "stackhpc/victoria"
+jobs:
+  tag:
+    runs-on: ubuntu-20.04
+    env:
+      naming-things-is-hard-repository: stackhpc/naming-things-is-hard
+    name: Automatic tagging for ${{ github.ref_name }} 🏷
+    steps:
+      - name: Github checkout 🛎 [${{ github.repository }}]
+        uses: actions/checkout@a12a3943b4bdde767164f792f33f40b04645d846
+        with:
+          fetch-depth: 0
+      - name: Configure repository 🛠
+        run: |
+          git remote rename origin stackhpc
```

Comment:
When running multiple shell commands, it's good practice to use `&&` to ensure that the whole step fails if the first command fails.